controllers/velero.go

```diff
@@ -357,6 +361,18 @@ func (r *DPAReconciler) customizeVeleroDeployment(dpa *oadpv1alpha1.DataProtecti
 		veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--resource-timeout=%v", resourceTimeoutString))
 	}
 
+	// check for default-snapshot-move-data parameter
+	snapshotMoveData := getSnapshotMoveDataValue(dpa)
+	if len(snapshotMoveData) > 0 {
+		veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--snapshot-move-data=%s", snapshotMoveData))
+	}
+
+	// check for default-volumes-to-fs-backup
+	defaultVolumesToFSBackup := getDefaultVolumesToFSBackup(dpa)
+	if len(defaultVolumesToFSBackup) > 0 {
+		veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--default-volumes-to-fs-backup=%s", defaultVolumesToFSBackup))
+	}
```

Comment:
```diff
	installDeployment := install.Deployment(veleroDeployment.Namespace,
		install.WithResources(veleroResourceReqs),
		install.WithImage(getVeleroImage(dpa)),
		install.WithAnnotations(podAnnotations),
		install.WithFeatures(dpa.Spec.Configuration.Velero.FeatureFlags),
		install.WithUploaderType(uploaderType),
		// last label overrides previous ones
		install.WithLabels(veleroDeployment.Labels),
		// use WithSecret false even if we have secret because we use a different VolumeMounts and EnvVars
		// see: https://github.com/vmware-tanzu/velero/blob/ed5809b7fc22f3661eeef10bdcb63f0d74472b76/pkg/install/deployment.go#L223-L261
		// our secrets are appended to containers/volumeMounts in credentials.AppendPluginSpecificSpecs function
		install.WithSecret(false),
		install.WithServiceAccountName(common.Velero),
+		install.WithDefaultSnapshotMoveData(dpa.spec...),
+		install.WithDefaultVolumesToFsBackup(dpa.spec...),
	)
```

was gonna say we try to use something like above.. but seems like those don't accept bool.. so the only way to use podTemplateOption is to create an array then pass that in, requiring exporting [PodTemplateOption](https://github.com/vmware-tanzu/velero/blob/main/pkg/install/deployment.go#L32C6-L32C23) in velero. Not ideal.

Another would've been to add new podTemplateOption func called `func (xxx) WithDefaultSnapshotMoveDataBool(option bool) podTemplateOption`.. also not ideal.

So all things considered this is ok if we're rushing and/or moving away from using podTemplateOptions alltogether..